flowback/comment/views.py

```diff
@@ -32,18 +47,28 @@ class FilterSerializer(serializers.Serializer):
         score__gt = serializers.IntegerField(required=False)
         score__lt = serializers.IntegerField(required=False)
 
-    class OutputSerializer(serializers.Serializer):
-        id = serializers.IntegerField()
-        author_id = serializers.IntegerField()
-        author_name = serializers.CharField(source='author.username')
-        author_profile_image = serializers.ImageField(source='author.profile_image')
-        parent_id = serializers.IntegerField(allow_null=True)
-        created_at = serializers.DateTimeField()
-        edited = serializers.BooleanField()
-        active = serializers.BooleanField()
-        message = serializers.CharField()
-        attachments = FileSerializer(source="attachments.filesegment_set", many=True, allow_null=True)
-        score = serializers.IntegerField()
+    class OutputSerializer(CommentOutputSerializer):
```

Comment:
Statement does nothing, delete or move CommentOutputSerializer contents to it,
If you delete it, change paginated_response serializer_class param to CommentOutputSerializer

Reply:
Removed and updated the argument to use the new import